### PR TITLE
Move MLFlow deployment away from operator

### PR DESCRIPTION
**Pull request by DavidSpek**
## Summary
Sets the annotation needed for MLFlow to visualize artifacts. Still need to build a new image with boto3 and an updated version of MLFlow.

### CODE REVIEW

File: mlflow/plural/recipes/mlflow-aws.yaml [hunk 1]
```diff
@@ -25,9 +25,5 @@ sections:
   items:
   - type: TERRAFORM
     name: aws
-  - type: TERRAFORM
-    name: aws-standalone
   - type: HELM
     name: mlflow
```
Comment (michaeljguarino) [inline mlflow/plural/recipes/mlflow-aws.yaml:29]: This means existing users with an mlflow install are going to have the operator uninstalled and this chart replacing it?

**Review by michaeljguarino — COMMENTED** [review]
We should take a look into the total number of people currently using mlflow.  A breaking cha

File: mlflow/plural/recipes/mlflow-aws.yaml (hunk 1 above)
Comment (DavidSpek) [reply mlflow/plural/recipes/mlflow-aws.yaml:29]: That is the idea. MLFlow standalone will be replaced by this, the operator is no longer needed either.

File: mlflow/terraform/aws/main.tf [hunk 2]
```diff
@@ -4,6 +4,55 @@ resource "kubernetes_namespace" "mlflow" {
...
+  source = "github.com/pluralsh/module-library//terraform/irsa-policy"
+  cluster_name   = var.cluster_name
+  role_name      = var.role_name
+  namespace      = var.namespace
+  serviceaccount = var.serviceaccount_name
+  policy_json    = data.aws_iam_policy_document.mlflow.json
+
+}
+
+data "aws_iam_policy_document" "mlflow" {
```
Comment (michaeljguarino) [inline mlflow/terraform/aws/main.tf:28]: the s3 buckets module should autocreate a policy for you

File: mlflow/terraform/gcp/main.tf [hunk 3]
```diff
@@ -4,6 +4,7 @@ resource "kubernetes_namespace" "mlflow" {
     labels = {
       "app.kubernetes.io/managed-by" = "plural"
       "app.plural.sh/name" = "mlflow"
+      "platform.plural.sh/sync-target" = "pg"
     }
   }
```
Comment (michaeljguarino) [inline mlflow/terraform/gcp/main.tf:9]: looks like gcp/azure aren't having their object stores provisioned?

Comment (DavidSpek) [reply mlflow/terraform/gcp/main.tf:9]: We also don't have recipes for GCP and Azure yet. Given the past experience with MLFlow now we should test those cloud providers rigorously before releasing those recipes.

**Pull request merged**